tests/fixtures/semantic_manifest_yamls/simple_semantic_manifest/semantic_models/bookings_source.yaml

```diff
@@ -94,6 +96,3 @@ semantic_model:
     - name: host
       type: foreign
       expr: host_id
-    - name: create_a_cycle_in_the_join_graph
-      type: primary
-      expr: guest_id
```

Comment:
This entity seems to exist specifically to create a cycle in the graph. Perhaps that is something we no longer care about in DSI, but I want to confirm that's the case.

Reply:
That is a good case to check, but I ended up creating a separate model to test that case specifically. The issue was that we test for the output of dimensions frequently, and this name made it confusing to view the results. That's currently in MF - do we have a test here that would benefit from it?

Reply:
I think having the test in MF is a good place as it seems to make more sense as a SL implementation check than a protocol requirement.